src/views/Home.jsx

```diff
@@ -1,12 +1,76 @@
 import './Home.css';
 import { SingleList } from '../components/SingleList.jsx';
+import { useState, useEffect } from 'react';
+import { createList } from '../api/firebase.js';
+import { useNavigate } from 'react-router-dom';
+
+export function Home({ data, userId, userEmail, setListPath }) {
+	const [newListName, setNewListName] = useState('');
+	const [newListSuccess, setNewListSuccess] = useState(null);
+	const [message, setMessage] = useState(null);
+
+	const navigate = useNavigate();
+
+	const handleChange = (e) => {
+		setNewListName(e.target.value);
+	};
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+		const currentLists = data.map((list) => {
+			return list.name;
+		});
+
+		//Check if the new list has a title that already exists in db
+		if (currentLists.indexOf(newListName) > -1) {
```

Comment:
I like how this works! I feel like another option could be to use `currentLists.includes(newListName)` but the result is probably about the same.

Reply:
Good point! I changed it to .includes() for consistency since we use that to check for a '/' in the list name.